cli/command/image/build/context.go

```diff
@@ -197,17 +210,29 @@ func getDockerfileRelPath(givenContextDir, givenDockerfile string) (absContextDi
 	if !isUNC(absContextDir) {
 		absContextDir, err = filepath.EvalSymlinks(absContextDir)
 		if err != nil {
-			return "", "", errors.Errorf("unable to evaluate symlinks in context path: %v", err)
+			return "", errors.Errorf("unable to evaluate symlinks in context path: %v", err)
 		}
 	}
 
 	stat, err := os.Lstat(absContextDir)
 	if err != nil {
-		return "", "", errors.Errorf("unable to stat context directory %q: %v", absContextDir, err)
+		return "", errors.Errorf("unable to stat context directory %q: %v", absContextDir, err)
 	}
 
 	if !stat.IsDir() {
-		return "", "", errors.Errorf("context must be a directory: %s", absContextDir)
+		return "", errors.Errorf("context must be a directory: %s", absContextDir)
+	}
+	return absContextDir, err
+}
+
+// getDockerfileRelPath uses the given context directory for a `docker build`
+// and returns the absolute path to the context directory, the relative path of
+// the dockerfile in that context directory, and a non-nil error on success.
```

Comment:
> returns the absolute path to the context directory, the relative path of the dockerfile in that context directory, and a non-nil error on success.

The comment is hard to understand for me.. "returns the relative path of the dockerfile in the context directory, and nil error on success"?